README.md

```diff
@@ -20,7 +20,8 @@ However, on-device debugging is hard, so the engine can also be compiled to run
  - Clone the repository
  - Change into the spade directory and change code as desired
  - Run `build-with-docker.sh`
-
+   
+this will produce the spade.uf2 file which you can flash to your sprig. 
```

Comment:
Suggest referring to the relative path where the spade.uf2 file is (be specific about exact location).